src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../moduls/User.js';
```

Comment:
There is a typo in the import path: 'moduls' should be 'modules'. Please correct the path to '../modules/User.js'.